paths/workflow_runs.yaml

```diff
@@ -0,0 +1,102 @@
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/workflow_runs/workflow_run.yaml
+        default:
+          $ref: ../responses/default_error.yaml
+
+    post:
+      summary: Create a workflow run.
+      operationId: create_workflow_run
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                workflow_id:
+                  type: string
+                  required: true
+                  description: The unique identifier for the workflow.
+                applicant_id:
+                  type: string
+                  required: true
+                  description: The unique identifier for the applicant.
+                custom_data:
+                  type: object
+                  description: Optional object with custom input data to be used in the workflow run.
+                link:
+                  type: object
+                  description: Optional object for the configuration of the workflow run link.
+      responses:
+        "201":
+          description: A workflow run object.
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/workflow_runs/workflow_run.yaml
+        default:
+          $ref: ../responses/default_error.yaml
+
+  task:
+    get:
+      summary: A single task can be retrieved by calling this endpoint with the unique identifier of the task and workflow run.
+      operationId: retrieve_task
+      parameters:
+        - name: workflow_run_id
+          in: path
+          required: true
+          description: The unique identifier of the workflow run to which the task belongs.
+          schema:
+            type: string
+        - name: task_id
+          in: path
+          required: true
+          description: The identifier of the task you want to retrieve.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A task object.
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/tasks/task.yaml
+        default:
+          $ref: ../responses/default_error.yaml
+
+  tasks:
+    get:
+      summary: The tasks of a workflow can be retrieved by calling this endpoint with the unique identifier of the workflow run.
+      operationId: list_tasks
+      parameters:
+        - name: workflow_run_id
+          in: path
+          required: true
+          description: The unique identifier of the workflow run to which the tasks belongs.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A subset of task objects that were already started or completed, ordered by the created_at field, in ascending order.
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/tasks/task.yaml
```

Comment:
I think we cannot do here a direct reference to the task resource since, on this endpoint, we only return a subset of it.

Reply:
also, I believe this should be an array of tasks, not a single one.